app/vmctl/influx.go

```diff
@@ -96,7 +97,7 @@ func (ip *influxProcessor) run(silent, verbose bool) error {
 	for err := range errCh {
 		return fmt.Errorf("import process failed: %s", err)
 	}
-	barpool.Stop()
+
```

Comment:
The change means the bar will be stopped after `Import finished` and Stats are printed. Does it break the output somehow? Have you tried to see how it looks in terminal?

Reply:
Stopping bar after `Import finished` will not break output, but having `barpool.Stop` called twice seems to be redundant.

Reply:
Ok